pennylane/templates/state_preparations/mottonen.py

```diff
@@ -12,8 +12,7 @@
 # See the License for the specific language governing permissions and
```

Comment:
Minor suggestion (not essential to change, feel free to resolve), but how about `templates/preparations`? I'm always slightly averse to underscores in my (import) paths :laughing:

Reply:
I would rather leave it, because the user won't use this path: recommended import it still  ``pennylane.templates.Mottonnen...``. But ``preparations`` may be confusing as to what gets prepared.